app/assets/stylesheets/darkswarm/embedded_shopfront.css.scss

```diff
@@ -51,6 +51,16 @@ nav.top-bar ul.right li.powered-by {
   }
 }
 
+.powered-by-embedded {
+   opacity: 0.6;
+   font-family: "Oswald", sans-serif;
```

Comment:
don't we inherit this already? If I'm not mistaken we use this typeface in other places. If that wasn't the case I'd go with a sass variable. thoughts?

Reply:
@sauloperez 

Cool. Agreed. This does exist as a sass variable already so i've used that instead. Have updated in latest version which I will push shortly.